src/content/structured/patterns/top-navigation-layout.mdx

```diff
@@ -369,9 +516,10 @@ For more information, see the [page header component](/components/page-header).
 
 export const snippetsTopNavPageHeader = [
   {
-    language: "Web component",
-    snippet:
-      createWebComponentsIndexHTML(`<ic-classification-banner></ic-classification-banner>
+    technology: "Web component",
```

Comment:
![Screenshot 2024-06-03 at 11 59 56](https://github.com/mi6/ic-design-system/assets/111292477/8ad29f9a-a71b-4071-98a0-f16a47530188)

There's an error with the Web Components code-block here. (React Codeblock displays fine)

Reply:
The Side navigation page has the same issue.